modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -80,6 +80,59 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-171637. Verify that an alert will appear when user don't fill a mandatory* field and that “Continue” button only will be clickable after fill all mandatory fields"
+	@priority = "5"
+	test CannotContinueWithoutFillRequiredField {
+		task ("Given I go to add a new MDF Request") {
+			PRMMarketing.gotoAddNewMDF();
+		}
+
+		task ("When I fill all mandatory fields on the MDF Request") {
+			PRMMDFRequest.fillGoalsPage(
+				accountName = "Account 1",
+				additionalOptions = "6.x to DXP Migration/Upgrade,Migration from competitor platform",
+				campaignDescription = "Campaign Description",
+				campaignName = "Campaign Name",
+				countryName = "Global",
+				goalsOptions = "Lead generation",
+				targetMarketOptions = "Education,Energy",
+				targetRolesOptions = "Administrator");
+		}
+
+		task ("Then the Continue button will be clickable") {
+			Button.click(button = "Continue");
+		}
+
+		task ("When I don't fill all mandatory fields on the MDF Request") {
+			PRMNavigator.goToNavItem(navItem = "Marketing");
+
+			PRMMarketing.gotoAddNewMDF();
+
+			PRMMDFRequest.fillGoalsPage(
+				accountName = "Account 1",
+				additionalOptions = "6.x to DXP Migration/Upgrade,Migration from competitor platform",
+				campaignName = "Campaign Name",
+				countryName = "Global",
+				goalsOptions = "Lead generation",
+				targetMarketOptions = "Education,Energy",
+				targetRolesOptions = "Administrator");
+		}
+
+		task ("Then the Continue button will be disabled") {
+			PRMMDFRequest.viewDisabledButton(buttonName = "Continue");
+		}
+
+		task ("When some mandatoryfield aren't fill") {
```

Comment:
Hey @MGabriellaS, just a typo here, is missing a space. Could you fix, please?

Reply:
Ok, I will fix.